aten/src/ATen/native/native_functions.yaml

```diff
@@ -2104,14 +2104,17 @@
 # `interpolation_mode` because it only supports Bilinear interpolation mode.
 # Nor does it take in `align_corners` because it only supports the mode
 # `align_corners = True`.
+#
+# `grid_sampler_2d_backward` takes in `output_mask` to optimize performance
```

Comment:
nit: move this comment to the corresponding function below.
Also I think this is a good place to note that this is not actually always followed: say that only the [0] value is checked and only on specific device(if that is true?).

Reply:
Thanks, moved the comment and made it more explicit that only `output_mask[0]` is checked. Both the CPU and CUDA implementations referred to in the dispatch-section for `grid_sampler_2d_backward` should respect the value of `output_mask[0]` (unless I'm missing something).